### ISSUE TITLE
build_region-reads_associations  Erros : Segmentation fault (core dumped) 

### ISSUE THREAD

**Issue by lookgene** [kensung-lab/SurVirus#12]
hello  
I would like to use Survirus to detect integration sites in human cancer samples.  but   i get an   core dumped   error  in  `build_region-reads_associations` step .  can you  help me . thanks for your help. 
 
```
(py2) [zhangqx@mu01 HPV]$ /home/zhangqx/project/HPV/lab/SurVirus/build_region-reads_associations /home/share/database/hg38/hg38.fa /home/share/database/HPV/HPV.fasta ./result/sample-202201-0177 ./result/sample-202201-0177/bam_0/
Segmentation fault (core dumped)
```

**Comment by Mesh89** [comment]
Hi,

Are you using publicly available data? Would I be able to test it?

**Comment by lookgene** [comment]
yes, I used public data [SRR1611127](https://www.ncbi.nlm.nih.gov/sra/?term=SRR1611127) to detect HPV integrate site in human genome hg38 。

**Comment by lookgene** [comment]
> Hi,
> 
> Are you using publicly available data? Would I be able to test it?

Would you  give a test  data ,i can fellow .

**Comment by Mesh89** [comment]
Thank you. I will try. Where did you obtain your HPV fasta?

**Comment by lookgene** [comment]
I download HPV reference sequence from this web site  https://pave.niaid.nih.gov/index.html#explore/reference_genomes/human_genomes , which are from NCBI
